pkgs/build-support/rust/cargo-local-registry.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchurl, stdenv }:
+
+let
+  version = "0.1.4";
+  platform =
+    if stdenv.system == "x86_64-linux"
```

Comment:
Or we could just build our own boostrap binary 🤷‍. Once the bootstrap is in place we can just grab a couple of versions back.